CHANGELOG.md

```diff
@@ -77,82 +79,8 @@ Co-authored-by: Sibin Grasic <sibin.grasic@oblak.studio>
 -   Bump minimatch and mocha by @dependabot in https://github.com/valeryan/vscode-phpsab/pull/84
 -   Bump nanoid and mocha by @dependabot in https://github.com/valeryan/vscode-phpsab/pull/85
 
-## New Contributors
+### New Contributors
 
 -   @stormrockwell made their first contribution in https://github.com/valeryan/vscode-phpsab/pull/78
 
 **Full Changelog**: https://github.com/valeryan/vscode-phpsab/compare/v.0.0.15...v.0.0.16
-
```

Comment:
I believe this was some historic change logs I brought over from the deprecated previous version of my extension. They are not really useful and I am pretty sure the first published version of this extension was 0.0.16